resources/META-INF/plugin.xml

```diff
@@ -21,6 +21,11 @@
 
   <change-notes>
     <![CDATA[
+0.1.8.1:
+<ul>
+  <li>improve handling of breakpoints for the bazel launch config</li>
```

Comment:
I just cherry-picked in two breakpoint fixes to create 0.1.8.1 (i.e., it doesn't contain all the commits from the last release).